index.js

```diff
@@ -50,3 +50,10 @@ function leftPad (str, len, ch) {
   // pad `str`!
   return pad + str;
 }
+
+function ecmaPadStart (str, len, ch) {
+  str = str + '';
+  if (len <= str.length) return str;
+  if (!ch && ch !== 0) ch = ' ';
```

Comment:
This is probably unnecessary as the definition for padStart states:

> If fillString is undefined, let filler be the String value consisting solely of the code unit 0x0020 (SPACE).

Reply:
Thanks for the information, but there seems to be a bug when using "str".padStart(10, null); which retruns 'nullnulstr' in node v 8.9.4, whereas this implementation returns: '       str'

Sorry for the late reply.

Reply:
(I reported a bug in the chromium project about this behavior and they pointed me to the spec where it is written that only undefined is ' ' (space). So this implementation should be retained due to backwards compatibility. 😅)